runtime/account_test.go

```diff
@@ -2595,111 +2595,6 @@ func TestRuntimeAccountLink(t *testing.T) {
 		assert.ErrorContains(t, err, "value of type `AuthAccount` has no member `linkAccount`")
 	})
 
-	t.Run("enabled, pragma", func(t *testing.T) {
```

Comment:
Should we keep this test, and only change the linking path from `public` to `private`?

Reply:
I started on it, then realized that the access part, which uses `getCapability`, would also have to be adjusted, i.e. the after the linking the capability must be stored somewhere, then retrieved in the access part.

After the refactor I realized that the "publish and claim" test case below does exactly that, so there was no need to keep this one here.